### PR TITLE
Add sass loader

### PR DESCRIPTION
**Pull request by eljam**
I think sass loader should be a default one with less if you want to cover all possibilities

### CODE REVIEW

File: src/Component/Configuration/Loader/SassLoader.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+
+        if (empty($config['filename'])) {
+            // If the filename is not set, apply inline style tags.
+            $code_blocks[] = (new CodeBlock())->set(CodeBlock::LOADER, '{ test: /\.scss$/, loader: \'style!css!sass\' }');
+        } else {
+            // If a filename is set, apply the ExtractTextPlugin
+            $fn            = 'fn_extract_text_plugin_sass';
+            $code_blocks[] = (new CodeBlock())
+                ->set(CodeBlock::HEADER, 'var '.$fn.' = require("extract-text-webpack-plugin");')
```
Comment (iltar) [inline src/Component/Configuration/Loader/SassLoader.php]: Some minor code style inconsistencies; Can you add spaces around the dots for concatenation?

File: README.md [hunk 2]
```diff
@@ -170,15 +171,15 @@ As mentioned in the example above, you may optionally specify a file type along
 type of file extension, just make sure you have the appropriate loaders enabled.
 
 For example, `js` will always work by default. However, `css` will only work if the `css-loader` is enabled. If you have
-the `less-loader` enabled, you can do something like this:
+the `less-loader` or `sass-loader` enabled, you can do something like this:
 
 ```twig
 <section>
-    {% webpack inline less %}
+    {% webpack inline less|sass %}
```
Comment (iltar) [inline README.md]: With `less|sass`, what would the result be?

Comment (haroldiedema) [inline README.md]: Agreed. This should (must) not be possible.

Webpack selects the correct loader, and only one loader, based on what you define there.

Comment (eljam) [inline README.md]: I was trying to say that you can use either less or sass in inline mode.
I will update the readme with the code style

**Comment by haroldiedema** [comment]
:+1:

**Comment by iltar** [comment]
Thanks @eljam!

**Comment by eljam** [comment]
Happy to help :)

**Pull request merged**